hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -438,6 +461,7 @@ private boolean doHoodieMetadataTableValidationOnce() {
       if (!cfg.ignoreFailed) {
         throw e;
       }
+      throwables.add(e);
```

Comment:
where are you clearing up the throwables list?
if we are calling doHoodieMetadataTableValidationOnce repeatedly, don't we need to clear the list before we execute next time ?